pkg/cosign/env/env_test.go

```diff
@@ -0,0 +1,214 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package env
+
+import (
+	"io"
+	"os"
+	"testing"
+)
+
+func TestGetenv(t *testing.T) {
+	os.Setenv(VariableExperimental.String(), "1")
```

Comment:
Nit: TBH for a test, I think it might be more readable to hardcode `COSIGN_EXPERIMENTAL`. It makes it clearer what you're testing.

I'll go with your judgment here, though. WYDT?